L/libcgal_julia/build_tarballs.jl

```diff
@@ -3,11 +3,15 @@
 using BinaryBuilder
 import Pkg: PackageSpec
 
+# reminder: change the above version if restricting the supported julia versions
+julia_versions = [v"1.6.3", v"1.7.0", v"1.8.0", v"1.9.0"]
+julia_compat = join("~" .* string.(getfield.(julia_versions, :major)) .* "." .* string.(getfield.(julia_versions, :minor)), ", ")
+
 name = "libcgal_julia"
-rversion = v"0.17.0"
+rversion = v"0.18.0"
 version = VersionNumber(rversion.major,
                         rversion.minor,
-                        100rversion.patch + julia_version.minor)
+                        100rversion.patch)
```

Comment:
If we're no longer encoding the julia version in the artefact version, we may as well do away with this.

```diff
- rversion = v"0.17.0"
- version = VersionNumber(rversion.major,
-                         rversion.minor,
-                         100rversion.patch + julia_version.minor)
+ version = v"0.18.0"